### ISSUE TITLE
python-libsbml installation fails on Python3.10

### ISSUE THREAD

**Issue by dweindl** [sbmlteam/libsbml#173]
python-libsbml installation fails on Python3.10 because the `formatter` module has been removed there:

```
[...]
    -- Build files have been written to: /tmp/pip-install-xl98ai6x/python-libsbml_9de198c7ca304e9285025543ef82614e/build/temp.linux-x86_64-3.10
    cmake --build . --target binding_python_lib --config Release --
    Scanning dependencies of target binding_python_swig
    [  0%] Swig Python source
    Traceback (most recent call last):
      File "/tmp/pip-install-xl98ai6x/python-libsbml_9de198c7ca304e9285025543ef82614e/libsbml_source/src/bindings/python/doc-converter/rewrite_pydoc.py", line 121, in <module>
        from formatter import NullWriter, AbstractFormatter
    ModuleNotFoundError: No module named 'formatter'
    gmake[3]: *** [src/bindings/python/CMakeFiles/binding_python_swig.dir/build.make:674: src/bindings/python/libsbml_wrap.cpp] Error 1
    gmake[2]: *** [CMakeFiles/Makefile2:370: src/bindings/python/CMakeFiles/binding_python_swig.dir/all] Error 2
    gmake[1]: *** [CMakeFiles/Makefile2:350: src/bindings/python/CMakeFiles/binding_python_lib.dir/rule] Error 2
    gmake: *** [Makefile:283: binding_python_lib] Error 2
    error: command '/usr/bin/cmake' failed with exit code 2
    Running setup.py install for python-libsbml ... error
```

From the [Python 3.10 release notes](https://docs.python.org/3/whatsnew/3.10.html):
> Removed the formatter module, which was deprecated in Python 3.4. It is somewhat obsolete, little used, and not tested. It was originally scheduled to be removed in Python 3.6, but such removals were delayed until after Python 2.7 EOL. Existing users should copy whatever classes they use into their code. (Contributed by Dong-hee Na and Terry J. Reedy in bpo-42299.)

**Comment by fbergmann** [comment]
thanks for letting me know, i'll be sure to get this solved and prepare a new pypi release for 3.10

**Comment by fbergmann** [comment]
it took a while longer, but new python binaries for 3.10 are now on pypi and conda
